requirements.txt

```diff
@@ -27,13 +27,13 @@ PyYAML>=6.0.0,<7.0.0
 rich>=13.3.1,<14.0.0
 rouge-score>=0.1.2,<0.2.0
 sentencepiece>=0.2.0,<0.3.0
-tokenizers>=0.15.2,<0.16.0
+tokenizers>=0.19.1,<0.20.0
```

Comment:
This break Intel Gaudi support. Is the version bump really required? Could you just add the higher version to the optional dependencies instead of main dependencies?